patchnotes/4.2.0.md

```diff
@@ -1 +1,32 @@
+## Patchnotes 4.2.0
```

Comment:
I don't want to have this redundant heading.

If it should say "patchnotes" explicitly in the release, I'd rather put it into the action code.

Reply:
I only see the version number to be redundant. I put it in there because the file itself should contain the version and not only in its name.

At least the "patchnotes" heading should stay in the file. Maybe the version number can be moved to a "footer"?

Reply:
We decided to keep the patchnotes heading incl. version number.